src/analysis/static/graph/CFG.ts

```diff
@@ -50,4 +51,70 @@ export class CFG {
       .filter((node) => node.type === NodeType.Root)
       .map((node) => <RootNode>node);
   }
+
+  getNodeById(nodeId : string): Node {
+    const found = this._nodes.filter((node: Node) => node.id == nodeId);
+    if(found.length != 1) {
+      console.log("No node with such id in CFG");
+      return null;
+    }
+    return found[0];
+  }
+
+  getRotatedAdjacencyList(): Map<String, Pair<string, number>[]> {
```

Comment:
Can you maybe at some documentation as to what the map exactly contains.
I'm guessing that the "second" in the pair is the weight?